packages/core/src/modules/proofs/formats/presentation-exchange/models/RequestPresentation.ts

```diff
@@ -0,0 +1,102 @@
+import type { PresentationExchangeProposalFormat } from '../../PresentationExchangeFormatsServiceOptions'
+import type { InputDescriptors } from './InputDescriptors'
+
+import { Expose } from 'class-transformer'
+import { IsOptional, IsString, ValidateNested } from 'class-validator'
+
+import { JsonTransformer } from '../../../../../utils'
+
+interface ProofTypeOptions {
+  proofType: string[]
+}
+
+export class ProofType {
+  public constructor(options: ProofTypeOptions) {
+    if (options) {
+      this.proofType = options.proofType
+    }
+  }
+  @Expose({ name: 'proof_type' })
+  public proofType!: string[]
+}
+
+// export class LdpVc {
```

Comment:
Commented code

Reply:
I have commented this code back in for now. This was @NB-PrasadKatkar's original code. I need to find out if it has any use.